helm/prometheus-agent/charts/prometheus-agent/templates/prometheus.yaml

```diff
@@ -47,8 +47,9 @@ spec:
         {{- if .Values.watchdog.command }}
         {{- toYaml .Values.watchdog.command | nindent 8 }}
         {{- else }}
-        - sh
-        - /etc/prometheus/configmaps/prometheus-agent-watchdog/watchdog.sh
+        - /bin/sh
+        - -c
+        - WATCHDOG_OVERLOAD_TOLERANCE=5 sh /etc/prometheus/configmaps/prometheus-agent-watchdog/watchdog.sh
```

Comment:
BTW could you make this a value that we could override via helm with a default of 5?